stock-filters/ChirpVillage/Biomes/BlockTypes.py

```diff
@@ -64,9 +65,10 @@
     'Acacia Door Block': (196, 0),
     'Dark Oak Door Block': (197, 0),
     'White Glazed Terracotta': (235, 0),
+    'Yellow Glazed Terracotta': (239, 0),
     'Gray Glazed Terracotta': (242, 0),
     'Black Glazed Terracotta': (250, 0)
 }
 
 # Blocks which sit ontop of the surface ie. Grass, Trees, Mushrooms etc
-surface_blocks = [0, 6, 8, 10, 17, 18, 31, 32, 37, 38, 39, 40, 59, 78, 81, 83, 99, 100, 103, 104, 105, 106, 111, 141, 142, 161, 162, 175]
+surface_blocks = [0, 6, 17, 18, 31, 32, 37, 38, 39, 40, 59, 78, 81, 83, 99, 100, 103, 104, 105, 106, 111, 141, 142, 161, 162, 175]
```

Comment:
why are we removing flowing water and flowing lava?

Reply:
I am removing them so that the surface is on top of the water rather then at the bottom. I found if we were placing the village on flowing water, the water was leaking all over our village.  I though this might help to fix that. 

A better solution is to write a clean up function that gets rid of trees, flowing water, etc but that will be alot more effort

Reply:
ahh valid! this works for now

Reply:
I can revert this if we prefer. I can see how i could get messy